en/appendices/4-0-upgrade-guide.rst

```diff
@@ -39,17 +39,18 @@ To help expedite fixing these tedious changes there is an upgrade CLI tool:
     cd upgrade
     git checkout 4.x
     composer install --no-dev
+    cd ..
 
 With the upgrade tool installed you can now run it on your application or
 plugin:
 
 .. code-block:: bash
 
     # Rename locale files
-    bin/cake upgrade file_rename locales <path/to/app>
+    upgrade/bin/cake upgrade file_rename locales <path/to/app>
```

Comment:
This is not how it should work

You need to browse/navigate into that app and run it relative from there.

Reply:
That is what I am saying. Do you think that the documentation makes that clear?

Reply:
But this change is doing the opposite

Reply:
What I mean is that you need to be in the application that you want to upgrade and not in `upgrade`. However this line `bin/cake upgrade file_rename locales <path/to/app>`suggests that you are in `upgrade` instead. (Also there is the `cd upgrade` line earlier.)
Maybe the documentation could explicitly say "You need to browse/navigate into that app and run it relative from there." and then something like `<path/to/upgrade>bin/cake upgrade file_rename locales <path/to/app>`

Reply:
No that is how it is supposed to be: inside upgrade app. Docs should be clear on this.

Reply:
But it is not possible to use `git mv` from inside the upgrade app on files from my app (like /src/Locale) which are in a different repository. That is when the ` fatal: ../src/Locale is outside repository` error occurs.

Reply:
The tool should itself cd into the working dir before doing that op. I am sure this is a small bug.

Reply:
I haven't seen this error before, what OS were you using?

Reply:
ah ok, I didnt see that it was supposed to change directory.
The reason for my problem was, that I gave the path to the app relative. (Sorry about that). But if the point of reference changes that can`t work.
Sorry for wasting your time with this, will close the PR now.